messaging/kafka-producer/src/test/java/io/quarkus/ts/messaging/kafka/producer/DevModeKafkaSnappyIT.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.ts.messaging.kafka.producer;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.DevModeQuarkusApplication;
+
+@QuarkusScenario
+public class DevModeKafkaSnappyIT extends SnappyCompressionIT {
+
+    @DevModeQuarkusApplication(properties = "devservices.properties")
+    static RestService app = new RestService()
+            .withProperty("quarkus.kafka.devservices.enabled", Boolean.TRUE.toString());
+
+    @Test
+    public void kafkaContainerShouldBeStarted() {
+        app.logs().assertContains("Creating container for image: docker.io/vectorized/redpanda:v24.1.2");
```

Comment:
Where is this image taken from? I don't like that assert checks also for version. Can't that one change independently from the tests?